tfx/components/trainer/executor.py

```diff
@@ -178,8 +178,8 @@ def Do(self, input_dict: Dict[Text, List[types.Artifact]],
 
     Args:
       input_dict: Input dict from input key to a list of ML-Metadata Artifacts.
-        - examples: Examples used for training, must include 'train' and 'eval'
-          splits.
+        - examples: Examples used for training, support custom splits. If not
+           specified, include 'train' and 'eval' splits by default.
```

Comment:
how about this[1] to be more specific.

[1] examples: Examples used for training, must include 'train' and 'eval' if custom splits is not specified in train_args and eval_args.

Reply:
yes, i think this looks more clear, i will update this